source/macos/AdaptiveCards/AdaptiveCards/Views/ACRContentStackView.swift

```diff
@@ -202,6 +215,65 @@ class ACRContentStackView: NSView, ACRContentHoldingViewProtocol, SelectActionHa
         }
     }
     
+    func increaseIntrinsicContentSize(_ view: NSView) {
+        let key = String(format: "%p", view)
+        subviewIntrinsicContentSizeCollection[key] = NSValue(size: view.intrinsicContentSize)
+    }
+    
+    func decreaseIntrinsicContentSize(_ view: NSView) {
+        // This Empty function design for override methods
+    }
+    
+    func updateIntrinsicContentSize() {
+        // This Empty function design for override methods
+    }
+    
+    func updateIntrinsicContentSize(_ block: @escaping (_ view: Any, _ idx: Int, _ stop: UnsafeMutablePointer<ObjCBool>) -> Void) {
+        (stackView.arrangedSubviews as NSArray).enumerateObjects(block)
+    }
+    
+    func getMaxHeightOfSubviews(afterExcluding view: NSView?) -> CGFloat {
+        return getViewWithMaxDimension(
+            afterExcluding: view,
+            dimension: { [self] vw in
+                var key: String?
+                if let vw = vw {
+                    key = String(format: "%p", vw)
+                }
+                let value = self.subviewIntrinsicContentSizeCollection[key ?? ""]
+                return (value != nil ? value?.sizeValue : .zero)?.height ?? 0.0
+            })
+    }
+    
+    func getMaxWidthOfSubviews(afterExcluding view: NSView?) -> CGFloat {
+        return getViewWithMaxDimension(
+            afterExcluding: view,
+            dimension: { [self] vw in
+                var key: String?
+                if let vw = vw {
+                    key = String(format: "%p", vw)
+                }
```

Comment:
Can we get on a call to understand this logic?

Reply:
yes sure.